LPD-46117 Add createItemURL to Item Selector Modal

SelenaAungst opened the pull request:
https://liferay.atlassian.net/browse/LPD-46117

Resent from https://github.com/liferay-platform-experience/liferay-portal/pull/1473

liferay-continuous-integration commented:
The following guidelines have been set by the owner of this repository: 
- &nbsp;&nbsp;&nbsp;&nbsp;"Thank you for sending this PR to liferay-platform-experience. If you haven't already,take this opportunity to review our [pull request guidelines](https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2915598365). For additional help,please reach out via the [appropriate channels](https://liferay.atlassian.net/wiki/spaces/PEDS/overview#%3Aslack%3A-Slack-Channels)."

SelenaAungst commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-31

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-31.liferay.com/job/test-portal-source-format/7725/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7725/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><p><span>Ran com.liferay.source.formatter at released version <a href="https://repository.liferay.com/nexus/content/repositories/liferay-public-releases/com/liferay/com.liferay.source.formatter/1.0.1534">1.0.1534</a>.</span><br/><em>*The <a href="https://github.com/liferay/liferay-portal/commits/master/modules/util/source-formatter">latest version</a> has not been released.</em></p><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/088f4c778bc91ca425b0b651fda14e00fd23bcef">088f4c778bc91ca425b0b651fda14e00fd23bcef</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/SelenaAungst/liferay-portal/tree/LPD-46117-4">LPD-46117-4</a><br/>Branch GIT ID: <a href="https://github.com/SelenaAungst/liferay-portal/commit/7faf1fd5de44ebcbfd006310d8a7d5f5ff7970df">7faf1fd5de44ebcbfd006310d8a7d5f5ff7970df</a></p>1 out of 1 jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7725/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7725/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
Test suite relevant has been triggered on http://test-1-28

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/14038/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14038/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 10 out of 11 jobs passed</h3><h3>:x: ci:test:relevant - 15 out of 19 jobs passed in 1 hour 54 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/1636">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1287401_14038</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/088f4c778bc91ca425b0b651fda14e00fd23bcef">088f4c778bc91ca425b0b651fda14e00fd23bcef</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/049bdc916e741c4551791bd651ecb0e154b75f4f">049bdc916e741c4551791bd651ecb0e154b75f4f</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/303004879">EE Development Acceptance (master) - 1615 - 2025-08-27[00:33:57]</a></p><details><summary><strong>ci:test:stable - 10 out of 11 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1347591/">functional-upgrade-tomcat101-postgresql163_stable/0/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1135369/">functional-upgrade-tomcat101-postgresql163_stable/0/1</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1317829/">functional-upgrade-tomcat101-postgresql163_stable/0/2</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203771/">functional-upgrade-tomcat101-postgresql163_stable/0/3</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1253772/">functional-upgrade-tomcat101-postgresql163_stable/0/4</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1221016/"><strike>js-unit/0/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1221017/">js-unit/1/0</a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1253769/">modules-integration-postgresql163_stable/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1426818/">modules-integration-postgresql163_stable/0/1</a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1253770/">playwright-js-tomcat101-postgresql163_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1243309/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>10 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 15 out of 19 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/14038/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1347591/">functional-upgrade-tomcat101-postgresql163_stable/0/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1135369/">functional-upgrade-tomcat101-postgresql163_stable/0/1</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1317829/">functional-upgrade-tomcat101-postgresql163_stable/0/2</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203771/">functional-upgrade-tomcat101-postgresql163_stable/0/3</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1253772/">functional-upgrade-tomcat101-postgresql163_stable/0/4</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1221016/"><strike>js-unit/0/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1221017/">js-unit/1/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1426820/">modules-integration-postgresql163/0/0</a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1253769/">modules-integration-postgresql163_stable/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1426818/">modules-integration-postgresql163_stable/0/1</a></li></ul></li><li>modules-semantic-versioning<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1316392/">modules-semantic-versioning/0/0</a></li></ul></li><li>modules-unit<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120776/">modules-unit/0/0</a></li></ul></li><li>playwright-js-tomcat101-mysql57<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1426819/">playwright-js-tomcat101-mysql57/0/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775/"><strike>playwright-js-tomcat101-mysql57/1/0</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770/"><strike>playwright-js-tomcat101-mysql57/1/1</strike></a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1253771/">playwright-js-tomcat101-mysql57/2/0</a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1253770/">playwright-js-tomcat101-postgresql163_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1243309/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>15 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14038/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1221016//consoleText">js-unit/0/0</a><ol><li><details><summary><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1221016/testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest">:apps:site:site-cms-site-initializer:packageRunTest</a></summary><div><ol><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1221016//testReport/(root)/:apps:site:site-cms-site-initializer:packageRunTest/apps_site_site_cms_site_initializer_test_js_structure_builder_components_picklist_builder_PicklistOptions_opens_the_add_option_modal_when_the_Add_New_button_from_the_management_toolbar_is_pressed">apps.site.site-cms-site-initializer.test.js.structure_builder.components.picklist_builder.PicklistOptions opens the add option modal when the Add New button from the management toolbar is pressed</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//consoleText">playwright-js-tomcat101-mysql57/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-09/test-1-28/test-portal-acceptance-pullrequest(master)/14038/playwright-js-tomcat101-mysql57/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775/testReport/frontend-data-set-admin-web_main_tests_data-set-admin_selectionFilters.spec/ts">frontend-data-set-admin-web/main/tests/data-set-admin/selectionFilters.spec.ts</a></summary><div><ol><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_selectionFilters.spec/ts/Can_create_a_selection_filter_with_API_Headless_source">Can create a selection filter with API Headless source</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_selectionFilters.spec/ts/Can_create_and_delete_a_selection_filter_from_picklist_source">Can create and delete a selection filter from picklist source</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_selectionFilters.spec/ts/Can_create_and_edit_a_selection_filter_with_API_Headless_source_using_composed_complex_fields">Can create and edit a selection filter with API Headless source using composed/complex fields</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_selectionFilters.spec/ts/Can_deactivate_and_activate_Selection_filters">Can deactivate and activate Selection filters</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_selectionFilters.spec/ts/Preselected_filter_values_are_checked_in_the_multiSelect">Preselected filter values are checked in the multiSelect</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775/testReport/frontend-data-set-admin-web_main_tests_data-set-admin_clientExtensionFilters.spec/ts">frontend-data-set-admin-web/main/tests/data-set-admin/clientExtensionFilters.spec.ts</a></summary><div><ol><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_clientExtensionFilters.spec/ts/Can_create_a_Client_Extension_Filter_in_DSM">Can create a Client Extension Filter in DSM</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_clientExtensionFilters.spec/ts/Can_deactivate_and_activate_a_Client_Extension_Filter_in_DSM">Can deactivate and activate a Client Extension Filter in DSM</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_clientExtensionFilters.spec/ts/Can_not_create_a_Client_Extension_Filter_in_DSM">Can not create a Client Extension Filter in DSM</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775/testReport/frontend-data-set-admin-web_main_tests_data-set-admin_dateRangeFilters.spec/ts">frontend-data-set-admin-web/main/tests/data-set-admin/dateRangeFilters.spec.ts</a></summary><div><ol><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_dateRangeFilters.spec/ts/Ability_to_save_and_edit_DSM_date_filters__LPS_181281">Ability to save and edit DSM date filters @LPS-181281</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_dateRangeFilters.spec/ts/No_date_filters_can_be_created_on_an_already_used_field">No date filters can be created on an already used field</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1120775//testReport/frontend-data-set-admin-web_main_tests_data-set-admin_dateRangeFilters.spec/ts/When_creating_a_new_filter_in_DSM__date_time_field_is_available_for_selection__LPD_10754">When creating a new filter in DSM, date-time field is available for selection @LPD-10754</a></li></ol></div></details></li><li>...</li></ol></div></li><li><div><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//consoleText">playwright-js-tomcat101-mysql57/1/1</a> - <a href="https://storage.cloud.google.com/testray-results/2025-09/test-1-28/test-portal-acceptance-pullrequest(master)/14038/playwright-js-tomcat101-mysql57/1/1/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770/testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_selectionFilters.spec/ts">frontend-data-set-admin-web/main/tests/data-set-fragment/selectionFilters.spec.ts</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_selectionFilters.spec/ts/Selection_filter_of_type__API_REST_Application__is_displayed_in_fragment__LPD_10754">Selection filter of type "API REST Application" is displayed in fragment @LPD-10754</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_selectionFilters.spec/ts/Selection_filter_of_type__API_REST_Application__with_a_composed_field_name_is_displayed_in_the_fragment">Selection filter of type "API REST Application" with a composed field name is displayed in the fragment</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_selectionFilters.spec/ts/Selection_filter_of_type__Object_Picklist__can_be_configured_to_include_or_exclude_selected_values">Selection filter of type "Object Picklist" can be configured to include or exclude selected values</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_selectionFilters.spec/ts/Selection_filter_of_type__Object_Picklist__is_displayed_in_fragment__LPD_10754">Selection filter of type "Object Picklist" is displayed in fragment @LPD-10754</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770/testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_sorting.spec/ts">frontend-data-set-admin-web/main/tests/data-set-fragment/sorting.spec.ts</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_sorting.spec/ts/When_sorting_is_configured_with_1_sort_that_it_is_inactive__the_dropdown_does_not_display_it_in_the_fragment">When sorting is configured with 1 sort that it is inactive, the dropdown does not display it in the fragment</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_sorting.spec/ts/When_sorting_is_configured_with_at_least_1_sort_and_it_is_active__the_dropdown_is_displayed_in_the_fragment">When sorting is configured with at least 1 sort and it is active, the dropdown is displayed in the fragment</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770/testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_pagination.spec/ts">frontend-data-set-admin-web/main/tests/data-set-fragment/pagination.spec.ts</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_pagination.spec/ts/Data_Set_Pagination_configuration_in_the_fragment___FDS_uses_custom_pagination_configuration_after_creating_a_Data_Set">Data Set Pagination configuration in the fragment › FDS uses custom pagination configuration after creating a Data Set</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1203770//testReport/frontend-data-set-admin-web_main_tests_data-set-fragment_pagination.spec/ts/Data_Set_Pagination_configuration_in_the_fragment___FDS_uses_default_pagination_configuration_after_creating_a_Data_Set">Data Set Pagination configuration in the fragment › FDS uses default pagination configuration after creating a Data Set</a></li></ol></div></details></li><li>...</li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14038/bundles/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a></li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/14038">test-portal-acceptance-pullrequest(master)#14038</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14038/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-platform-experience/liferay-portal/pull/1479">liferay-platform-experience#1479</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/310370216">[master] ci:test:relevant - SelenaAungst &gt; liferay-platform-experience - PR#1479 - 2025-09-05[17:19:14]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>310370216</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/14038/">test-portal-acceptance-pullrequest(master)#14038</a><br/></div></div>

SelenaAungst commented on modules/apps/frontend-js/frontend-js-item-selector-web/src/main/resources/META-INF/resources/item_selector/itemSelectorModal.tsx:
```diff
@@ -16,6 +16,11 @@ import React, {useEffect, useState} from 'react';
+	/**
+	 * URL for item creation used to open a new modal.
+	 */
+	createItemURL: string;
```
I think it is optional, @ethib137 can you confirm?

ethib137 replied:
Yes that's correct. It should be optional.

SelenaAungst replied:
Are there any specific changes I need to make in this case? @gabsprates

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-33

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-33.liferay.com/job/test-portal-source-format/4292/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/4292/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><p><span>Ran com.liferay.source.formatter at released version <a href="https://repository.liferay.com/nexus/content/repositories/liferay-public-releases/com/liferay/com.liferay.source.formatter/1.0.1535">1.0.1535</a>.</span><br/><em>*The <a href="https://github.com/liferay/liferay-portal/commits/master/modules/util/source-formatter">latest version</a> has not been released.</em></p><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b8ffc3ea27fa379a103881d9d97110ffafb33ca8">b8ffc3ea27fa379a103881d9d97110ffafb33ca8</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/SelenaAungst/liferay-portal/tree/LPD-46117-4">LPD-46117-4</a><br/>Branch GIT ID: <a href="https://github.com/SelenaAungst/liferay-portal/commit/3c0bef2e908d2504e8ecd65212407950c328f17d">3c0bef2e908d2504e8ecd65212407950c328f17d</a></p>1 out of 1 jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-33.liferay.com/job/test-portal-source-format/4292/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/4292/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-33.liferay.com/job/test-portal-source-format/4292">test-portal-source-format#4292</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/4292/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-platform-experience/liferay-portal/pull/1479">liferay-platform-experience#1479</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/316059731">[master] ci:test:sf - SelenaAungst &gt; liferay-platform-experience - PR#1479 - 2025-09-12[19:16:34]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>316059731</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-33.liferay.com/job/test-portal-source-format/4292/">test-portal-source-format#4292</a><br/></div></div>

SelenaAungst commented:
@ethib137 I added your changes and fixed some styling. It looks like I need to rebase so I will resend this PR tomorrow. I also am still looking into the testing issues.
